pkg/metrics/ntp.go

```diff
@@ -113,9 +120,8 @@ func (p *Ntp) Offset() (time.Duration, error) {
 		total += offset
 	}
 
-	// cache the value to be reused
-	p.offset = total / time.Duration(len(offsets))
-	p.updatedAt = p.now()
+	offsetSeconds := (total / time.Duration(len(offsets))).Seconds()
+	p.offset = &offsetSeconds
```

Comment:
from what I understood, `p.offset` is not needed anymore, isn't it?

Reply:
Totally right, fixed in https://github.com/newrelic/infrastructure-agent/pull/1699/commits/61841480c20f60b597795180dcac6aa40f4793ab

Thanks!